device.yml

```diff
@@ -98,7 +98,7 @@ registers:
     address: 52
     type: U8
     access: Write
-    maskType: PwmPlaybackMode
```

Comment:
Nice catch! I like the PwmPlaybackMode as it future proofs in case we add other playback modes in the future, do you mind changing the bitmask instead?

Reply:
This is a multi PWM board... We'll never going to have a TonePlaybackMode :-)
Anyway, I prefer to keep it like this since to avoid retesting the package.

Reply:
Fair enough :P